src/prefabs/structures/Box/options/index.ts

```diff
@@ -145,6 +147,13 @@ export const boxOptions = {
       },
     },
   ),
+  zIndex: number('Z-index', {
+    value: '0',
```

Comment:
The default z-Index should be set to `auto` by default: https://developer.mozilla.org/en-US/docs/Web/CSS/z-index
But because this is a number input, `auto` won't be a valid value.
I'm thinking it might be nice to set this to an empty string by default, and then (in src/components/box) add an extra if statement:
```javascript
if (zIndex === '') {
  return auto;
}
```
This way we can still make use of a number option, but still fall back to 'auto' if no value is filled in :)